lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -233,16 +233,16 @@ module Scope
           column_valid_to = "#{table_name}.valid_to"
           column_valid_from = "#{table_name}.valid_from"
           ignore_valid_datetime
-            .tap { |relation| break relation.where("? <= #{column_valid_to}", from.in_time_zone.to_datetime) if from }
-            .tap { |relation| break relation.where("#{column_valid_from} <= ?", to.in_time_zone.to_datetime) if to }
+            .tap { |relation| break relation.where(klass.arel_table[:valid_to].gteq(from.in_time_zone.to_datetime)) if from }
```

Comment:
If so, I think `column_valid_to`  and `column_valid_from` are unnecessary.

Reply:
Thanks! 
I removed it. https://github.com/kufu/activerecord-bitemporal/pull/35/commits/ea4d585fe11ea8f2f1ef772918f2c5e3ebbf47ae